### DISCUSSION TITLE
mistake

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20218] [General]
mistake

![Screenshot_2024-07-01-17-08-36-013_net osmand plus](https://github.com/osmandapp/OsmAnd/assets/53188843/75531664-5957-4339-9be6-54d4d0f286d6)

**Comment by DmitryAlexei** [comment]
Users can modify translations using https://hosted.weblate.org/projects/osmand/

**Comment by ZeljkoBG** [comment]
You didn't look at the picture carefully.

The profile is Auto (Driving) - the profile icon is a car. So there are 4 wheels.
The type is Motorcycle - and a motorcycle has 2 wheels.

See image from version 4.7xxx. And the translation is the same.

![4 7](https://github.com/osmandapp/OsmAnd/assets/53188843/1cf69515-5e8a-4497-b2c2-66c03eaed5f8)

The developer made a mistake

**Comment by dmpr0** [comment]
All profile settings can be changed: icon and type.

**Reply by ZeljkoBG** [reply]
By default it is an error. I installed last nightly version and it's working fine now. Fixed without "... can be changed: icon and type."

**Reply by DmitryAlexei** [reply]
@ZeljkoBG you can change routing type in navigation settings. See docs
https://osmand.net/docs/user/navigation/routing/car-based-routing

**Reply by ZeljkoBG** [reply]
Dear Dmitry, my English is poor, as soon as you don't understand my remark. Now everything is fine. But it has nothing to do with settings. To which you refer me.
I'll try again. If you don't understand me, then nothing, because everything is fine now.

1. I install (clean install) the nightly version, **from Sunday.** 
I don't download maps. I turn off the internet. I choose the DRIVING profile as the basic one. I have a car icon and this text:
**Driving
Type: Motorcycle**

2.  I install (clean install) the nightly version, **from Tuesday.** 
I don't download maps. I turn off the internet. I choose the DRIVING profile as the basic one. I have a car icon and this text:
**Driving
Type: Car**


Bottom line: if you see a difference, it's not up to me. There was a problem with the application.

**Reply by DmitryAlexei** [reply]
You can customize your driving profile by changing your navigation type or even creating your own routing via modifying and importing routing.xml.
Navigation type can be changed in navigation > settings > navigation settings >  navigation type
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/679ef056-bb6f-4e42-99c9-a17451240008"  width="400" height="auto"/> </td>
  </tr>
</table>

**Reply by ZeljkoBG** [reply]
Thank you. I understand. Very strange to me. I've never seen anything like it. And so much experience and beta testing on 7-8 applications...
Instead of simply selecting the Motorcycle profile, I can select DRIVING and then motorcycle... or ...
Too complicated. But OK.
